lib/ansible/module_utils/vmware.py

```diff
@@ -380,6 +380,14 @@ def gather_vm_facts(content, vm):
         else:
             mac_addr = mac_addr_dash = None
 
+        if (hasattr(entry, 'backing') and hasattr(entry.backing, 'port') and
```

Comment:
```
        port_group_key = port_key = None
        if (hasattr(entry, 'backing') and hasattr(entry.backing, 'port') and
                hasattr(entry.backing.port, 'portKey') and hasattr(entry.backing.port, 'portgroupKey')):
            port_group_key = entry.backing.port.portgroupKey
            port_key = entry.backing.port.portKey
```

Reply:
Logically the same. But I have two issues about that:
1) I would rather not mutate the variables
2) The code right above mine uses the same if-then-else pattern, so I tried keeping it consistent

Let me know if you'd rather me change it, though.

Reply:
I like @Akasurde's feedback, but will not push for it, it's a detail between code consistency and less code. @Akasurde has the final call :)